help/_posts/1970-01-01-AOSP.md

```diff
@@ -2,7 +2,6 @@
 layout: help
 category: help
 mirrorid: AOSP
-permalink: /help/AOSP/
```

Comment:
Maybe keep these filenames and `permalink` as-is? @shankerwangmiao

Reply:
I think these redundant attributes confusing, which also render the inconsistent file name useless.